src/App.js

```diff
@@ -1,5 +1,78 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+function printResult(sum){
+  return MissionUtils.Console.print(`결과 : ${sum}`);
+}
+
+function sumNumber(numberArr){
+  let sum = 0;
+  for(var number of numberArr) sum += number;
+  return sum;
+}
+
+function extractNumber(input, splitunit){
+  if(input.indexOf("\\n") != -1)
+    input = input.substring(input.indexOf("\\n")+2, input.length);
+
+  const regex = new RegExp(`[${splitunit}]`);
+  let numberArr = input.split(regex).map(Number);
+  return numberArr;
```

Comment:
```suggestion
   return input.split(regex).map(Number);
```
이렇게 축약하거나 `numberArr`를 선언할 때 `const` 키워드로 선언해도 좋을 것 같아요!